controllers/cluster/applier/applier.go

```diff
@@ -77,36 +77,38 @@ func (a *Applier) ReconcileCluster(infra *v1.Infra, hosts []v1.Hosts, cluster *c
 			return fmt.Errorf("execute ssh commend fail:%v", err)
 		}
 	}
+
 	for i := 0; i < len(node); i++ {
 		ip := node[i]
-		if err = scpCluster(ssh, ip, infra.Spec.SSH, "node"); err != nil {
+		if err = scpCluster(ssh, ip, cluster.Spec.SSH, "node"); err != nil {
 			return fmt.Errorf("scp cluster node error :%v", err)
 		}
-
 		err = ssh.CmdAsync(ip, "sudo sh nodeDepl.sh node"+strconv.Itoa(i)+" "+passwd, "")
 		if err != nil {
 			return fmt.Errorf("execute ssh commend fail:%v", err)
 		}
 	}
-	if err = scpCluster(ssh, sealos, infra.Spec.SSH, "master"); err != nil {
+	if err = scpCluster(ssh, sealos, cluster.Spec.SSH, "master"); err != nil {
 		return fmt.Errorf("scp cluster master error :%v", err)
 	}
 	ipMasters, ipNodes := getIPs(master), getIPs(node)
 
-	fmt.Println("sudo sh masterDepl.sh master0 " + passwd + " " + ipMasters + " " + ipNodes)
-	//todo Add images
-	err = ssh.CmdAsync(sealos, "sudo sh masterDepl.sh master0 "+passwd+" "+ipMasters+" "+ipNodes, "")
+	fmt.Println("sudo sh masterDepl.sh master0 " + passwd + " " + ipMasters + " " + ipNodes + " " + images)
+	err = ssh.CmdAsync(sealos, "sudo sh masterDepl.sh master0 "+passwd+" "+ipMasters+" "+ipNodes+" "+images, "")
 	if err != nil {
 		return fmt.Errorf("execute ssh commend fail:%v", err)
 	}
-
 	return nil
 }
 
 func getImages(cluster *cv1.Cluster) string {
 	images := ""
-	for _, image := range cluster.Spec.Images {
-		images = images + image + " "
+	for i, image := range cluster.Spec.Images {
```

Comment:
I don't quite understand why need this

Reply:
array -> string(as parameter; split:",") -> array(in shell) -> string(split: " ")

Reply:
here: array->string, to be parameter